soh/soh/Enhancements/randomizer/3drando/fill.cpp

```diff
@@ -1039,6 +1039,19 @@ int Fill() {
     //Fast fill for the rest of the pool
     std::vector<uint32_t> remainingPool = FilterAndEraseFromPool(ItemPool, [](const auto i) { return true; });
     FastFill(remainingPool, GetAllEmptyLocations(), false);
+
+    // Add prices for scrubsanity
```

Comment:
might be nice to elaborate on why we need to do this for soh (since it's not happening in 3drando), even just replacing the current comment with
```cpp
// in soh, we read scrub item prices from the spoilerfile
```
would probably do the trick imo